app/view_models/thredded/group_view.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module Thredded
+  # A view model for a page of PostViews.
+  class GroupView
```

Comment:
nit: a more specific name such as `MessageboardsGroupView` or `GroupMessageboardsView` would be better. Also, the comment.